extensions/amp-bind/0.1/bind-expr-impl.jison

```diff
@@ -282,9 +282,11 @@ primitive:
         // Use JSON.parse() to process special chars e.g. '\n'.
         // JSON doesn't recognize single-quotes, so use double-quote in
         // leading/trailing chars and escape double-quote in the string.
-        const string = yytext.substr(1, yyleng - 2);
-        const parsed = tryParseJson(`"${string.replace(/"/g, '\\"')}"`);
-        this.$ = new AstNode(AstNodeType.LITERAL, null, parsed || string);
+        const raw = yytext.substr(1, yyleng - 2);
+        // Since we accept escaped quotation marks, unescape them here.
+        const unescaped = raw.replace(/\\'/g, `'`).replace(/\\"/g, `"`);
```

Comment:
These can be joined into a single regex replace.

Reply:
Done